### PR TITLE
[4.0] Rename Control Panel

### PR DESCRIPTION
**Pull request by brianteeman**
The administration home area really isn't a control panel anymore.

This PR renames it to "Home"

### CODE REVIEW

**Comment by brianteeman** [comment]
The system test failure is correct - if accepted the test will need updating

File: administrator/language/en-GB/en-GB.plg_quickicon_extensionupdate.sys.ini [hunk 1]
```diff
@@ -4,4 +4,4 @@
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_QUICKICON_EXTENSIONUPDATE="Quick Icon - Joomla! Extensions Updates Notification"
-PLG_QUICKICON_EXTENSIONUPDATE_XML_DESCRIPTION="Checks for updates of your installed third-party extensions and notifies you when you visit the Control Panel page."
+PLG_QUICKICON_EXTENSIONUPDATE_XML_DESCRIPTION="Checks for updates of your installed third-party extensions and notifies you when you visit the administrator area home page.""
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.plg_quickicon_extensionupdate.sys.ini]:
```suggestion
PLG_QUICKICON_EXTENSIONUPDATE_XML_DESCRIPTION="Checks for updates of your installed third-party extensions and notifies you when you visit the administrator area home page."
```

File: administrator/language/en-GB/en-GB.mod_quickicon.ini [hunk 2]
```diff
@@ -25,4 +25,4 @@ MOD_QUICKICON_STRUCTURE="Structure"
 MOD_QUICKICON_TEMPLATE_MANAGER="Templates"
 MOD_QUICKICON_USER_MANAGER="Users"
 MOD_QUICKICON_USERS="Users"
-MOD_QUICKICON_XML_DESCRIPTION="This module shows Quick Icons that are visible on the Control Panel (administrator area home page)."
+MOD_QUICKICON_XML_DESCRIPTION="This module shows Quick Icons that are visible on the administrator area home page."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.mod_quickicon.ini:28]: Maybe change `administrator area` to `administration` to match `"Shows the Joomla! Administration home page."`

File: administrator/templates/atum/language/en-GB/en-GB.tpl_atum.ini [hunk 3]
```diff
@@ -12,7 +12,7 @@ TPL_ATUM_LOGIN_LOGO_LABEL="Login Logo"
 TPL_ATUM_LOGIN_PAGE_SETTINGS_LABEL="Login Page Settings"
 TPL_ATUM_MENU="MENU"
 TPL_ATUM_SITENAME_LABEL="Site name"
-TPL_BACK_TO_CONTROL_PANEL="Back to Control Panel"
+TPL_BACK_TO_HOME="Back to the administrator home page"
```
Comment (Quy) [inline administrator/templates/atum/language/en-GB/en-GB.tpl_atum.ini]: Sort in alpha order.

File: administrator/language/en-GB/en-GB.mod_menu.ini [hunk 4]
```diff
@@ -75,6 +74,7 @@ MOD_MENU_HELP_SUPPORT_OFFICIAL_LANGUAGE_FORUM_VALUE="511"
 MOD_MENU_HELP_SUPPORT_OFFICIAL_LANGUAGE_FORUM="Official Language Forums"
 MOD_MENU_HELP_TRANSLATIONS="Joomla! Translations"
 MOD_MENU_HELP_XCHANGE="Stack Exchange"
+MOD_MENU_HOME="Home"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.mod_menu.ini:79]: Is it necessary to create this with the same value as the next line?

Comment (brianteeman) [reply administrator/language/en-GB/en-GB.mod_menu.ini:79]: Not in english but as they serve different purposes they might in another language

**Comment by infograf768** [comment]
One of these useless changes which are not needed.

**Comment by brianteeman** [comment]
Please stop with the attacks

**Comment by C-Lodder** [comment]
Agree with the change. Personally I'd suggest using "Dashboard" over "Home", as I feel it's more of a widely used term on admin panels, but that's just me.

**Comment by simbus82** [comment]
Why "Home"? Other CMS/Ecommerce use "Dashboard", I agree that "control panel" is wrong.

**Comment by TobsBobs** [comment]
+1 for Dashboard

**Comment by jeckodevelopment** [comment]
I'd support "Dashboard" instead of Home

**Comment by brianteeman** [comment]
Currently it's not much of a dashboard. E-commerce systems do have much more dashboard funvtionality

**Pull request closed without merging**